upgrade_test.py

```diff
@@ -776,8 +776,6 @@ def _run_stress_workload(self, workload_name: str, wait_for_finish: bool = False
         InfoEvent(message=f"Starting {workload_name}").publish()
         stress_before_upgrade = self.params.get(workload_name)
         workload_thread_pool = self.run_stress_thread(stress_cmd=stress_before_upgrade)
-        if self.params.get('alternator_port'):
-            self.pre_create_alternator_tables()
```

Comment:
@Orenef11 so we don't need to pre_create these tables at all?

Reply:
No, we do it in this section
https://github.com/scylladb/scylla-cluster-tests/blob/92a0ac988a3e6ac9c5a81984339385d74bdb1912/longevity_test.py#L177-L195

This line is a duplicate.

Reply:
It looks like test gets a YCSB error:
```
18:11:39  < t:2021-11-18 16:11:38,629 f:file_logger.py  l:85   c:sdcm.sct_events.file_logger p:INFO  > 4575 [Thread-68] ERROR site.ycsb.db.DynamoDBClient  -com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException: User not found: alternator (Service: AmazonDynamoDBv2; Status Code: 400; Error Code: UnrecognizedClientException; Request ID: null; Proxy: null)
```

Reply:
@yarongilor 
Thanks for the comment.
Now, I think the issue is fixed
https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/Oren/job/rolling-upgrade-alternator-test/3/console